Azure/azure-cli
# {Network} `az network private-endpoint`: Added Device Update for IoT Hub support to Network Private Endpoint

**Related command**
No new command. Added Device Update for IoT Hub support to Network Private Endpoint

**Description**
Added Microsoft.DeviceUpdate/accounts support to Network Private Endpoint

**Testing Guide**
See the added test and recording

---

This checklist is used to make sure that common guidelines for a pull request are followed.
